{{cookiecutter.project_slug}}/.env.example

```diff
@@ -51,7 +50,7 @@ STAFF_EMAIL='{{cookiecutter.author_name}} <{{cookiecutter.email}}>'
 #     CURRENT_PORT      The port linked to the 'primary' domain for the purpose of user-facing
 #                           links.
 #
-ALLOWED_HOSTS='localhost,127.0.0.1'
+ALLOWED_HOSTS='server,localhost,127.0.0.1'
```

Comment:
Docker compose networking knows the backend server by the name `server`. So that needs to be added here to make Django happy.